Probesys/bileto
# sec: Hide errors details when reseting a password

## Related issue(s)



N/A

## How to test manually



1. go to the reset password form
2. enter an email that doesn't exist in the database
3. verify that Bileto tells you that an email has been sent if the email is correct

## Checklist
